data/persons.txt

```diff
@@ -699,7 +699,6 @@ ship "Modified Osprey"
 	"final explode" "final explosion large" 1
 
 
-
```

Comment:
Was this intentional?
Most gaps before `person` are three lines, only this one and one other (and the new patrol team) use two lines.

Reply:
I've no idea, actually. Maybe just a simple oversight over the years?